src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin clockSize($size) {
+  height: $size;
+  width: $size;
+}
+
+@mixin dotSize($size) {
+  height: $size;
+  width: $size;
+}
```

Comment:
This mixin makes the same as the previous one.
If it creating square, name it square, if it creating circle name it circle and add border-radius property

Reply:
Fixed